### ISSUE TITLE
.gitignore is missing in generated app

### ISSUE THREAD

**Issue by fpauser** [ember-cli/ember-octane-blueprint#14]
Somehow the included `files/.gitignore` is not copied over to the generated app. Thus the first commit contains all the normally .gitignored-stuff (e.g. node_modules).

**Comment by ppcano** [comment]
I cannot reproduce this error in `master`, the [.gitignore](https://github.com/ember-cli/ember-octane-blueprint/blob/master/packages/%40ember/octane-app-blueprint/.gitignore) file is copied correctly.

```
ember new my-octane-app -b ./ember-octane-blueprint/packages/\@ember/octane-app-blueprint/
```


Please, would you mind to try `master` again and inform us about the outcome.

**Comment by sheriffderek** [comment]
With just `ember new app-name -b ember-octane-app-blueprint` - I was also without ignore.

On your suggested command: 

![screen shot 2019-02-05 at 6 58 59 pm](https://user-images.githubusercontent.com/2066617/52318227-47dd2380-2978-11e9-902d-fe6b7bfff7f2.png)
Did I type it wrong?

**Comment by NullVoxPopuli** [comment]
@sheriffderek the command in the screenshot should only work if you've cloned this repo.

but, is .gitignore generated with just the classic `ember new classic-app`?

**Comment by sheriffderek** [comment]
Classic `ember new app-name` definitely creates a gitignore. (double checked)
`ember new app-name -b ember-octane-app-blueprint` does not (in my case) (ran it twice)

**Comment by NullVoxPopuli** [comment]
Well, that's good that this is a confirmable bug for multiple people!
I'll see what I can do!

**Comment by ppcano** [comment]
@NullVoxPopuli 

I thought that running `ember new app-name -b ember-octane-app-blueprint` is using the Pre-official packages which are not up-to-date. If this is the case, I think you should also update the Pre-official packages, so new users can start using the latest changes.

@sheriffderek 

The `Unknown blueprint` error could be that the blueprint path is not available. If you have not cloned the repo, you should do it and run as 

```
ember new my-app -b $pathToBlueprintRepo/ember-octane-blueprint/packages/\@ember/octane-app-blueprint
```

**Comment by NullVoxPopuli** [comment]
>  which are not up-to-date

they are auto-deployed. the only reason why they're not official is because they're published by my npm user instead of the `@ember` org.  Not sure if phrasing could be made more clear or not.

**Comment by ppcano** [comment]
@NullVoxPopuli 

You are right, I assumed wrongly the package were not updated. It is weird when using the blueprint local path, the `.gitignore` is copied but it will not be copied when using the npm package  😬

**Comment by NullVoxPopuli** [comment]
I wonder if I accidentally added it to npmignore, or maybe I need to
specifically add it to the files entry in the package.json?

**Comment by ppcano** [comment]
This issue has been fixed by #34 and can be closed.
